9oct auto switch

JimmyTanUWA opened the pull request:
this is the real and sim switch, but on auto mode

JimmyTanUWA committed:
Update receiver.py

added automatic switcher

JimmyTanUWA committed:
Update app.py

added automatic switcher

JimmyTanUWA committed:
Update config.py

added automatic switcher

IIEnat reviewed (changes requested):
I was there with u in the basement but idk where this went wrong:
- make_processors()
- start_processor_thread()
Those 2 functions are missing and so can't run app.py

JimmyTanUWA committed:
Update receiver.py

mistake in last commit

JimmyTanUWA committed:
Map draw issues

Station names are reverted
Making circle radius a config constant that tunes the frontend circle size.

JimmyTanUWA committed:
Update app.py

JimmyTanUWA commented:
oct 10 fixes bugs spotted by Ray where simulated stations weren't drawn to the map

IIEnat reviewed (approved):
Can confirm it works now. Thanks!
